src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -56,14 +56,14 @@ const defaultProps = {
 const OptionRowLHN = (props) => {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
 
-    React.useEffect(() => {
-        ReportActionContextMenu.hideContextMenu(false);
-    }, [optionItem.isPinned]);
-
     if (!optionItem) {
         return null;
     }
 
+    React.useEffect(() => {
```

Comment:
useEffect is not allowed after conditional early return.

This is what I suggest:
```
    const isPinned = Boolean(optionItem && optionItem.isPinned);
    React.useEffect(() => {
        ReportActionContextMenu.hideContextMenu(false);
    }, [isPinned]);

    if (!optionItem) {
        return null;
    }
```